libbeat/scripts/Makefile

```diff
@@ -51,7 +51,7 @@ build: $(GOFILES)
 # Create test coverage binary
 .PHONY: buildbeat.test
 buildbeat.test: $(GOFILES)
-	go test -c -covermode=atomic -coverpkg ${GOPACKAGES_COMMA_SEP}
+	go test -c -race -covermode=atomic -coverpkg ${GOPACKAGES_COMMA_SEP}
```

Comment:
I already made the `-race` off by default and it can be enabled via a`RACE_DETECTOR=1` environment variable. See #789 or the commit https://github.com/andrewkroh/beats/commit/0aac4045e7d57a28e251d7599f3d72987b7a4e79.